src/expr/proof_checker.cpp

```diff
@@ -107,6 +107,11 @@ Node ProofChecker::check(
     Assert(expected.isNull() || expected == args[0]);
     return expected;
   }
+  if (id == PfRule::VERIT_RULE)
```

Comment:
This shouldn't be here, let's move this to theory/builtin/proof_checker.cpp.

Reply:
Done